energy-growers/src/components/AccordionList/Accordion/Accordion.scss

```
@@ -0,0 +1,47 @@
+@import "../../../assets/styles/globals.scss";
+
+.eg-container-accordion {
+  border-bottom: rem(1) solid map-get($colors, light-grey);
+
+  .eg-accordion-item-container {
+    overflow: hidden;
+    transition: all 0.3s ease-in-out;
+  }
+  .eg-content-accordion {
+    @include remove-button-background;
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    margin: rem(24) rem(24) rem(24) rem(0);
+    .eg-question-accordion {
+      color: map-get($colors, dark-aqua);
+      text-align: start;
+      font-weight: 700;
```

```suggestion
      font-weight: 700;
      
      &.eg-open-accordion {
        color: map-get($colors, custom-pink);
        transition: all 0.3s ease-in-out;
      }

```

gli important non si usano, utilizziamo scss per annidare correttamente le classi